python/fastui/enums.py

```diff
@@ -0,0 +1,34 @@
+from enum import Enum
+
+
+class StrEnum(str, Enum):
```

Comment:
I'd rather not add all this, instead just use `class DisplayMode(str, Enum):`

Reply:
Ok, I copied the code from python 3.11 StrEnum.